test/unit/recipes/default_spec.rb

```diff
@@ -24,20 +22,20 @@
   let(:chef_run) { chef_runner.converge(described_recipe) }
   let(:node) { chef_runner.node }
 
-  %w(
+  %w[
     dovecot::ohai_plugin
     dovecot::user
     dovecot::from_package
     dovecot::conf_files
     dovecot::service
-  ).each do |recipe|
+  ].each do |recipe|
     it "includes #{recipe} recipe" do
       expect(chef_run).to include_recipe(recipe)
     end
   end # each recipe
 
   xcontext 'with install from source (not yet implemented)' do
-    before { node.set['dovecot']['install_from'] = 'source' }
+    before { node.normal['dovecot']['install_from'] = 'source' }
```

Comment:
even though these are just tests, node.normal shouldn't really be used here, it should most likely just be node.default, or node.override if it has to be that way...

Reply:
I just did a straightforward replace as per deprecation documentation here: https://chef.github.io/chef-rfc/rfc082-deprecation-policy.html

It says `node.set` must be replaced by `node.normal` which is an alias.

I will evaluate `node.default`

Reply:
yeah i wrote that deprecation =)

the "We highly suggest you evaluate your usage of node.set / node.normal though to see if that is actually the attribute behavior you desire" bit probably needs to be rewritten a little less passively and more aggressively....  that's sort of the point which is to stop using the normal level accidentally and convert to default/override...

Reply:
Probably this should be `override`, yes.